src/components/Suggestions/styled.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+const UList = styled.ul`
+  flex: 1;
+  padding: 0px;
+  list-style-type: none;
+`;
```

Comment:
This one works just fine, you can use more expressive names, maybe something like `VideoList` or 'SuggestionContainer`, choosing a name that tells the reader about the purpose of the component improves the readability